### PR TITLE
LPS-122148 Filter browsing content

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/contents/components/PageContents.js [hunk 1]
```diff
@@ -19,8 +19,31 @@ import Collapse from '../../../../../common/components/Collapse';
 import PageContent from './PageContent';
 import SearchContents from './SearchContents';
 
+export const CONTENT_TYPE_LABELS = {
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/contents/components/PageContents.js:21]: If we are using this in another component, should we move it to constants? 😛 I think it's what we usually do

**Review by sandrodw3 — COMMENTED** [review]
Looks great to me! 👏🏽😄 I just left a super tiny comment

**Comment by veroglez** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/contents/components/PageContents.js (hunk 1 above)
Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/contents/components/PageContents.js]: Done!

**Comment by victorg1991** [comment]
ci:reevaluate:1077401_7124

**Pull request closed without merging**